packages/opentelemetry-api/src/context/propagation/getter.ts

```diff
@@ -14,6 +14,11 @@
  * limitations under the License.
  */
 
-export type Carrier = {
-  [key: string]: unknown;
-};
+export type GetterFunction<Carrier = any> = (
+  carrier: Carrier,
+  key: string
+) => unknown;
+
+export function defaultGetter(carrier: any, key: string): unknown {
```

Comment:
Please include JSDoc comment, also worth to mention that it will return `null` if they key doesn't exist.

Reply:
It won't return `null` if the property doesn't exist. It will return undefined.